src/cloud-api-adaptor/podvm/Makefile

```diff
@@ -84,15 +84,21 @@ $(IMAGE_FILE): $(BINARIES) $(FILES) setopts
 	mkdir -p toupload
 	if [ "${SE_BOOT}" = "1" ] && [ "${ARCH}" = "s390x" ]; then \
 		qemu-img create -f qcow2 "se-${IMAGE_FILE}" 100G; \
+		# Temporary workaround for installing cryptsetup on RHEL 9.4 and below s390x base images for enabling se \
+		# Due to issue: https://gitlab.com/qemu-project/qemu/-/issues/2054 \
+		# Remove this if using the latest QEMU version (v9.0.0) \
+		if [ "${PODVM_DISTRO}" = "rhel" ]; then \
+			yum install -y cryptsetup; \
+			cp /usr/sbin/cryptsetup ./files; \
```

Comment:
copying directly to ./files/usr/bin/ will allow drop the copy later, no?

Reply:
i think we need binaries on /usr/bin so this may not drop later copy.

Reply:
i think it's copied by https://github.com/confidential-containers/cloud-api-adaptor/blob/3f67b4e7deef4469f9997dc6380d3082691b4921/src/cloud-api-adaptor/podvm/qcow2/copy-files.sh#L25

Reply:
oh! My bad, updated the code per the suggestions. Thank you so much. 
Could you please re-review